packages/base/card-api.gts

```diff
@@ -1465,9 +1470,33 @@ class IDCard extends Card {
   };
 }
 
+export class StringCard extends CardBase {
+  static [primitive]: string;
+  static [useIndexBasedKey]: never;
+  static embedded = class Embedded extends Component<typeof this> {
+    <template>
+      {{@model}}
+    </template>
+  };
+  static edit = class Edit extends Component<typeof this> {
+    <template>
+      <BoxelInput @value={{@model}} @onInput={{@set}} />
+    </template>
+  };
+}
+
+export class Card extends CardBase {
+  @field title = contains(StringCard, {
+    computeVia: function (this: Card) {
+      return 'Untitled';
```

Comment:
We discussed possibly making this a normal field, not a computed, since it's already legal to override that with a computed anyway when you want that.

I think that's probably still a good idea. It doesn't change the case for child cards that want to override with a computed, and it means that other ones that want a manual title would have that (which they cannot otherwise do unless we add a storage API for computeds).

Reply:
I will make the change, but I'll note that I discovered it is currently also "legal" to override a computed with a non-computed.